prebid-video/video-long-form.md

```diff
@@ -141,6 +141,84 @@ A string indicating the type of content being displayed in the video player. The
 
 After you’ve defined your ad units, you can continue with the rest of your configuration.
 
+#### Deal Support
+
+To prioritize certain video deals and ensure delivery and performance within the Freewheel stack Prebid has additional configurations that can be enabled.
+
+`prioritizeDeals`  
+A  boolean that indicates if deals should be given a higher preference. If true, Prebid will set the value of `hb_pb_cat_dur` with `bid.video.tier` replacing the cpm value. For example:
+
+  <pre>
+    hb_pb_cat_dur=dealId_395_15s
+  </pre>
+
+`dealTier`  
+An object that enables publishers to set a prefix and minimum deal tier for each bidder. The `dealTier` object enables publishers to have different line item setups with varying priorities.
+
+`dealTier.prefix`  
+A string that enables bidders to target deal line items.
+
+`dealTier.minDealTier`  
+An integer that will give higher preference to  deal bids when the `minDealTier` value is greater than five (5). Bids with `minDealTier` values less than five will not be ignored, however their cache key will contain `dealId` in place of `cpm`. These bids will be auctioned as non-deal bids.
```

Comment:
Here 5 is looking like some hard coded value. Maybe something like this could be done

`An integer that will give higher preference to deal bids which return tier greater than minDealTier. `

The second part of para can be used in conjunction with the code sample below to explain the workings